src/mmw/apps/modeling/tasks.py

```diff
@@ -56,15 +58,17 @@ def start_histogram_job(self, json_polygon):
     kick off the SJS job to generate a histogram
     of the provided polygon (i.e. AoI).
     Returns the id of the job so we can poll later
-    for the results, as well as the pixel width of
+    for the results, the pixel width of
     the polygon so that areas can be calculated from
-    the results.
+    the results, and the input json_polygon to use to
+    retrieve animal population data from the db.
     """
     polygon = json.loads(json_polygon)
 
     return {
         'pixel_width': aoi_resolution(polygon),
-        'sjs_job_id': histogram_start([json_polygon], self.retry)
+        'sjs_job_id': histogram_start([json_polygon], self.retry),
+        'aoi': json_polygon
```

Comment:
I'm still seeing `json_polygon` being returned here, when it doesn't need to be.